File: Documentation/4.0/Samples/csharp/Raven.Documentation.Samples/ClientApi/Session/HowTo/IgnoreChanges.cs
using Raven.Client.Documents;
using Raven.Documentation.Samples.Orders;

namespace Raven.Documentation.Samples.ClientApi.Session.HowTo
{
    public class IgnoreChanges
    {
        private interface IFoo
        {
            #region ignore_changes_1
            void IgnoreChangesFor(object entity);
            #endregion
        }

        public IgnoreChanges()
        {
            using (var store = new DocumentStore())
            {
                using (var session = store.OpenSession())
                {
                    #region ignore_changes_2
                    Product product = session.Load<Product>("products/1-A");
                    session.Advanced.IgnoreChangesFor(product);
                    product.UnitsInStock += 1; //this will be ignored for SaveChanges
                    session.SaveChanges();
                    #endregion

                }

            }
        }
    }
}

File: Documentation/5.4/Samples/python/ClientApi/Session/HowTo/IgnoreChanges.py
from examples_base import ExampleBase, Product


class Foo:
    # region ignore_changes_1

    def ignore_changes_for(self, entity: object) -> None: ...

    # endregion


class IgnoreChanges(ExampleBase):
    def setUp(self):
        super().setUp()

    def test_ignore_changes(self):
        with self.embedded_server.get_document_store("IgnoreChanges") as store:
            with store.open_session() as session:
                self.add_products(session)
                # region ignore_changes_2
                product = session.load("products/1-A", Product)
                session.ignore_changes_for(product)
                product.units_in_stock += 1  # this will be ignored for save_changes
                session.save_changes()
                # endregion
